decred/decred/dcr/account.py

```diff
@@ -314,21 +322,7 @@ def blob(utxo):
     def unblob(b):
         """Satisfies the encode.Blobber API"""
         ver, d = encode.decodeBlob(b)
-        if ver == 0:
-            if len(d) != 7:
-                # TODO change to DecredError
-                raise AssertionError(
-                    "wrong number of pushes for TicketInfo. wanted 7, got %d" % len(d)
-                )
-            d.extend([b"", b"", b"", b""])
```

Comment:
I suppose it's not so important at this stage, but this was intended to update the database. Once we have users, we can't expect them to delete their wallet and restore from the seed phrase every time we update something.

Reply:
made an issue #102

Reply:
Oh, I'm sorry, I missed that. Definitely an important feature that deserves its own consideration, and a clear and explicit implementation. :slightly_smiling_face: Thanks for making an issue about it.